### ISSUE TITLE
Does not work with latest Windows 10 SDK

### ISSUE THREAD

**Issue by GuildMasterInfinite** [nabijaczleweli/rust-embed-resource#5]
Hello, I tried to install the [cargo-update](https://github.com/nabijaczleweli/cargo-update) crate and ran into an issue. I believe it is caused by this crate, and the way it handles finding RC.exe.

```
--- stderr
thread 'main' panicked at 'Are you sure you have RC.EXE in your $PATH?:
Error { repr: Os { code: 2, message: "The system cannot find the file specified." } }', 
src\libcore\result.rs:860
```

The reason I don't have RC.exe in my `$PATH` is because since VS 2017, you can have multiple installed versions of VS 2017 on the same machine, and they are **not** added to the path anymore.

Now, I do have VS 2017 installed, and the C++ build tools, and Rust nightly does work and uses the VS 2017 linker. Is there a way for this crate to support VS 2017 too?

The [solution for Rust](https://github.com/rust-lang/rust/issues/38584) was to just update the [gcc](https://github.com/alexcrichton/gcc-rs) crate, which supports VS 2017 out of the box.

A temporary solution I found is: open the "x64 Native Tools Command Prompt for VS 2017", and run `cargo install` from there. But, again, it would be great if this crate worked with VS 2017 without special preparation.

<hr/>

Some more information:
[Finding installed Visual C++ tools for Visual Studio 2017](https://blogs.msdn.microsoft.com/vcblog/2017/03/06/finding-the-visual-c-compiler-tools-in-visual-studio-2017/)

**Comment by nabijaczleweli** [comment]
I look for `RC.EXE` in various places [here](https://github.com/nabijaczleweli/rust-embed-resource/blob/258a7bd188365b2d3ea7dba535f110329179c7fd/src/windows_msvc.rs#L23); it's probably possible to look for the 2017 version there, too. I'll see to it.

**Comment by nabijaczleweli** [comment]
After trying to read the shitty `vswhere` source (which I assume I can't just use because it's not a native VS binary) and attempting to comb through `gcc-rs`' -- I don't know how either of them do it. I'm guessing it's somewhere down the registry rabbit-hole, but where - I know not. Pointers would be appreciated.

**Comment by GuildMasterInfinite** [comment]
Couldn't `gcc-rs` function [`find_tool()`](https://docs.rs/gcc/0.3.51/gcc/windows_registry/fn.find_tool.html) be used? It seems to work for `cl.exe`, it should work for `rc.exe` too.

**Comment by GuildMasterInfinite** [comment]
I've found the problem - `embed-resource` probes the `<Windows SDK>/bin/x86` and `/x64` folders.

But on the newest Windows SDK, the executable has been moved to `/bin/10.0.15063.0/x64/`.

**Comment by nabijaczleweli** [comment]
I'd be grateful if you tested the referenced commit; I tried to do my best but there's only so far I can go without an actual test env for this.

**Comment by GuildMasterInfinite** [comment]
It worked! Thanks.

**Comment by nabijaczleweli** [comment]
Does it really work? I'm really surprised. I'll release with patch bump so all crates that depend on this one get the update automatically. Thanks for the help!

**Comment by nabijaczleweli** [comment]
Released in [`v1.1.3`](https://github.com/nabijaczleweli/rust-embed-resource/releases/tag/v1.1.3).
